service/controller/cluster.go

```diff
@@ -56,20 +77,29 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
 		return nil, microerror.Maskf(invalidConfigError, "%T.K8sClient must not be empty", config)
 	}
 
-	resourceSets, err := newClusterResourceSets(config)
+	resources, err := newClusterResources(config)
 	if err != nil {
 		return nil, microerror.Mask(err)
 	}
 
 	var operatorkitController *controller.Controller
 	{
 		c := controller.Config{
-			K8sClient:    config.K8sClient,
-			Logger:       config.Logger,
-			ResourceSets: resourceSets,
+			InitCtx: func(ctx context.Context, obj interface{}) (context.Context, error) {
```

Comment:
Not totally sure what this is. Can it be deleted?

Reply:
I think it is deprecated but would be something to discuss if `GuestUpdateEnabled` is still honored anywhere. It can be set in the installations repo but I think barely anyone knows.

Reply:
Checking on this.

Reply:
I researched this a bit. `kvm-operator` was the only operator that used it. It was enabled for every installation. No one disables it or has any plans to so I'm dropping it.